sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -221,6 +228,8 @@ object BasicWriteJobStatsTracker {
   private val NUM_OUTPUT_BYTES_KEY = "numOutputBytes"
   private val NUM_OUTPUT_ROWS_KEY = "numOutputRows"
   private val NUM_PARTS_KEY = "numParts"
+  private val DURATION_OF_TASK_COMMIT = "taskCommitDuration"
+  private val DURATION_JOB_COMMIT = "jobCommitDuration"
```

Comment:
Shall we rename these `vals` consistently?
- `DURATION_OF_TASK_COMMIT` -> `TASK_COMMIT_DURATION`
- `DURATION_JOB_COMMIT` -> `JOB_COMMIT_DURATION`?